surfinBH/_fit_evaluators/fit_7dq4Emri.py

```diff
@@ -0,0 +1,267 @@
+import numpy as np
+from surfinBH import surfinBH
+import warnings
+
+#=============================================================================
+class Fit7dq4Emri(surfinBH.SurFinBH):
+    """ A class for the NRSur7dq4EmriRemnant model presented in Boschini et al.,
+    arxiv:[TO_BE_CHANGED] (hereafter THE PAPER). 
+    
+    This model predicts the final mass mf and final spin chif, for the remnants 
+    of precessing binary black hole systems,  extending to arbitrarily large 
+    mass ratios.
+    The fits are done using Gaussian Process Regression (GPR) and also 
+    provide an error estimate along with the fit value.
+
+    This model has been trained in the parameter space:
+        NR: q <= 4, |chiA| <= 0.8, |chiB| <= 0.8
+        EMRI: 100 <= q <= 1000 , |chiA| <= 1, |chiB| <= 1
+
+    It can be evaluated at arbitrary mass ratios.
+
+    =========================================================================
+    Usage:
+
+    import surfinBH
+
+    # Load the fit
+    fit = surfinBH.LoadFits('NRSur7dq4EmriRemnant')
+
+    We provide the following call methods:
+        # remnant mass and 1-sigma error estimate
+        mf, mf_err = fit.mf(q, chiA, chiB, **kwargs)
+
+        # remnant spin and 1-sigma error estimate
+        chif, chif_err = fit.chif(q, chiA, chiB, **kwargs)
+        
+        # All of these together (this fit returns vf = vf_err = None)
+        # This output is needed for compatibility with the other fits
+        mf, chif, vf, mf_err, chif_err, vf_err = 
+            fit.all(q, chiA, chiB, **kwargs)
+
+    The arguments for each of these call methods are as follows:
+    Arguments:
+        q:     Mass ratio (q = mA/mB >=1)
+
+        chiA:  Dimensionless spin vector of the heavier black hole at
+               reference epoch.
+
+        chiB:  Dimensionless spin vector of the lighter black hole at
+               reference epoch.
+
+                This follows the same convention as LAL, where the spin
+                components are defined as:
+                \chi_z = \chi \cdot \hat{L}, where L is the orbital angular
+                    momentum vector at the epoch.
+                \chi_x = \chi \cdot \hat{n}, where n = body2 -> body1 is the
+                    separation vector at the epoch. body1 is the heavier body.
+                \chi_y = \chi \cdot \hat{L \cross n}.
+                These spin components are frame-independent as they are defined
+                using vector inner products. This is equivalent to specifying
+                the spins in the coorbital frame at the reference epoch. 
+                For comparable-mass binaries the reference epoch is assumed to 
+                be at t=-100 M from the peak of the waveform, which corresponds 
+                to 2-4 GW cycles before the peak. In the EMRI limit the 
```

Comment:
EMRI limit --> EMRI limit (q>=XX)
For XX, use the lowest q for the EMRI training data. 1000? I forgot.

Reply:
Done